core/src/test/java/org/elasticsearch/index/mapper/DynamicMappingDisabledIT.java

```diff
@@ -0,0 +1,29 @@
+package org.elasticsearch.index.mapper;
+
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.test.ESIntegTestCase;
+
+import java.io.IOException;
+
+/**
+ * Created by davidgalbraith on 12/17/15.
+ */
+public class DynamicMappingDisabledIT extends ESIntegTestCase {
+
+    @Override
+    protected Settings nodeSettings(int nodeOrdinal) {
+        return Settings.builder().put(super.nodeSettings(nodeOrdinal))
+            .put("index.mapper.dynamic", "false")
```

Comment:
Use the factored out [constant](https://github.com/elastic/elasticsearch/pull/15424/files#r47975685) here.